Physicc/include/collider.hpp

```diff
@@ -173,8 +181,12 @@ namespace Physicc
 			return Type::Box;
 		}
 
+		inline static std::vector<glm::vec4> getVertices()
```

Comment:
We want `ZoneScoped` in getter functions too. The reason being that if, after profiling, we see that it is taking unnecessarily long, then we can correct it. (It can take longer than expected if we return by value instead of doing a move, for example.)

Reply:
Right, I'll add that.

Reply:
I'll add it to the remaining functions without ZoneScoped as well, such as getCentroid, getType, getStaticType.